pkg/agent/nodeportlocal/npl_agent_test.go

```diff
@@ -761,22 +773,18 @@ var (
 	portTakenError = fmt.Errorf("Port taken")
 )
 
-// TestNodePortAlreadyBoundTo validates that when a port is already bound to, a different port will
-// be selected for NPL.
+// TestNodePortAlreadyBoundTo validates that when a port with TCP protocol is already bound to,
+// the same port should be selected for NPL if any other protocol is available.
```

Comment:
this comment is incorrect.

> the same port should be selected for NPL

we don't use the same port

Reply:
Updated.